core/src/main/scala/org/apache/spark/sql/rapids/tool/ToolUtils.scala

```diff
@@ -317,6 +317,20 @@ object SQLMetricsStats {
   }
 }
 
+object IgnoreExecs {
+  val AdaptiveSparkPlan = "AdaptiveSparkPlan" // AdaptiveSparkPlan is not a real exec. It is
+  // a wrapper for the whole plan.
```

Comment:
nit: Can we put the comment above variable definition for better scalastyle (fits in a single line). Similar change for `CollectLimit`?

```suggestion
  // AdaptiveSparkPlan is not a real exec. It is a wrapper for the whole plan.
  val AdaptiveSparkPlan = "AdaptiveSparkPlan"
```